### PR TITLE
Leaderboard Styles for Narrow and Copy to Clipboard

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Review by 0xBigBoss — COMMENTED** [review]
I don't think it makes sense to have referral code on this page for each send leader. 

It should have the profile avatars.

**Comment by youngkidwarrior** [comment]
Good call

File: packages/app/utils/NameDisplayUtils.ts
Comment (0xBigBoss) [inline packages/app/utils/NameDisplayUtils.ts:1]: we have this already. https://github.com/0xsend/sendapp/blob/01-19-Leaderboard_Styles_for_Narrow_and_Copy_to_Clipboard/packages/app/utils/strings.ts#L1

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Jan 25, 6:38 PM**: [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/76) rebased this pull request as part of a merge.

**Review by 0xBigBoss — DISMISSED** [review]
Looks good, but you should generate a snapshot for this screen so we know when we change it again.

File: packages/app/features/leaderboard/screen.tsx [hunk 1]
```diff
@@ -102,38 +105,44 @@ function LeaderboardSection() {
...
+      <Paragraph mb="0" size="$9" lineHeight="$4" fontWeight={'bold'}>
         {user.rank}
       </Paragraph>
-      <Paragraph w="20%" f={1} mb="0" size="$6" lineHeight="$4" ta="center">
-        {user.sendTag}
-      </Paragraph>
-      <Paragraph w="20%" f={1} mb="0" size="$6" lineHeight="$4" ta="center">
+      <XStack f={1} gap="$4" ai="center" jc={'center'}>
+        <XStack w="90%" maw="$20" gap="$1" ai="center">
+          <Image
```
Comment (0xBigBoss) [inline packages/app/features/leaderboard/screen.tsx:131]: Should use the `Avatar` component, plus it's not required to have an avatar URL so we should use a fallback component. https://tamagui.dev/docs/components/avatar/1.0.0#avatarfallback

File: packages/app/features/leaderboard/screen.tsx [hunk 2]
```diff
@@ -102,38 +105,44 @@ function LeaderboardSection() {
...
+            source={{ uri: user.profile.avatar_url }}
+            $gtMd={{ w: '$5', h: '$5' }}
+            w={'$4'}
+            h={'$4'}
+            br={9999}
+          />
+          <Paragraph f={1} mb="0" size="$6" lineHeight="$4" ta="left">
+            {media.gtMd ? shorten(user.sendTag, 12, 3) : shorten(user.sendTag, 6, 3)}
```
Comment (0xBigBoss) [inline packages/app/features/leaderboard/screen.tsx:139]: Nice...

**Review by 0xBigBoss — APPROVED** [review]

**Pull request closed without merging**